docs/manage/_get_support.rst

```diff
@@ -57,6 +57,18 @@ Open the ``.kolibri`` folder inside the :ref:`Home <home>` folder of the device
 
 .. warning:: On Linux and MacOS systems you will need to activate the *Show Hidden Files* option in your file browser, in order to view the ``.kolibri`` folder.
 
+Troubleshoot Database Issues
+----------------------------
+
+In case you receive the ``database disk image is malformed`` error in Terminal, try running these commands (note that **you must have the** ``sqlite3`` **command available on your system**).
+
+  .. code-block:: bash
+
+    sqlite3 /home/<your_username>/.kolibri/db.sqlite3 .dump | sqlite3 fixed.db 
```

Comment:
You can replace `/home/<your_username>/` with `~/` -- the syntax of this command will not work in a Windows shell anyways. It's not a given that the `.kolibri` folder is in `/home/`, it can also be in `/var/kolibri/`

Reply:
Good point @benjaoming, will change it asap! 👍